.github/CODEOWNERS

```diff
@@ -96,6 +96,9 @@
 # PRLabel: %Cognitive - Text Analytics
 /sdk/textanalytics/ @deyaaeldeen @witemple-msft
 
+# PRLabel: %Schema Registry
+/sdk/schemaregistry/ @deyaaeldeen
```

Comment:
@nguerrera, would you like to be in here too? :)

Reply:
Sure, I can if you would like me to! I'm happy to help review changes.

Reply:
Yeah I ask for your review in pretty much every SR PR :D, I opened https://github.com/Azure/azure-sdk-for-js/pull/16991.